x/feemarket/keeper/eip1559.go

```diff
@@ -25,13 +25,13 @@ func (k Keeper) CalculateBaseFee(ctx sdk.Context) *big.Int {
 
 	// If the current block is the first EIP-1559 block, return the InitialBaseFee.
 	if ctx.BlockHeight() == params.EnableHeight {
-		return new(big.Int).SetInt64(params.InitialBaseFee)
+		return k.GetBaseFee(ctx)
 	}
 
 	// get the block gas used and the base fee values for the parent block.
 	parentBaseFee := k.GetBaseFee(ctx)
```

Comment:
no need to get the params from the store again, it's already available and we just need to convert it to `big.Int`